lib/ddtrace/sampling/rule_sampler.rb

```diff
@@ -19,13 +19,13 @@ class RuleSampler
       attr_reader :rules, :rate_limiter, :default_sampler
 
       # @param rules [Array<Rule>] ordered list of rules to be applied to a span
-      # @param rate_limit [Float] number of traces per second, defaults to no rate limit
+      # @param rate_limit [Float] number of traces per second, defaults to +100+
       # @param rate_limiter [RateLimiter] limiter applied after rule matching
       # @param default_sample_rate [Float] fallback sample rate when no rules apply to a span,
       #   between +[0,1]+, defaults to +1+
       # @param default_sampler [Sample] fallback strategy when no rules apply to a span
       def initialize(rules = [],
-                     rate_limit: nil,
+                     rate_limit: 100,
```

Comment:
I think it might be a good idea to bump this into a constant, perhaps in the same class. Maybe `DEFAULT_RATE_LIMIT`?

Reply:
sure, makes sense